keygen/openssl.conf

```diff
@@ -1,4 +1,41 @@
 [req]
 distinguished_name = req_distinguished_name
+x509_extensions = v3_ca	# The extentions to add to the self signed cert
 
 [req_distinguished_name]
+
+[ v3_ca ]
+
+# Extensions for a typical CA
+
+
+# PKIX recommendation.
+
```

Comment:
Too many empty lines. You can write: "PKIX recommendation - extensions for a typical CA". That would be one line.